pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc

```diff
@@ -10,7 +10,7 @@ include::../attributes.adoc[]
 ifdef::context[:parent-context: {context}]
 
 [id="assembly-help_{context}"]
-= Pantheon Help Page
+= {ProductName} User Guide
```

Comment:
We're not publishing this document in Pantheon 2, right? I'm not 100 % sure atm. If yes, then we cannot have attributes in the titles.

Reply:
@dvagnero We are *not* publishing the EUD in Pantheon 2 (at the moment, at least). I'm going to leave this as-is.